release-process.md

```diff
@@ -71,45 +99,15 @@ Also check that all build and test passes are green from the RISELab Jenkins: ht
 Note that not all permutations are run on PR therefore it is important to check Jenkins runs.
 
 
-The process of cutting a release candidate has been partially automated via the RISELab Jenkins. There are
-Jenkins jobs that can tag a release candidate and create various packages based on that candidate.
-At present the Jenkins jobs *SHOULD NOT BE USED* as they use a legacy shared key for signing.
+To cut a release candidate, there are 4 steps:
+1. Create a git tag for the release candidate.
+1. Package the release binaries & sources, and upload them to a staging SVN repo.
+1. Create the release docs, and upload them to a staging SVN repo.
+1. Publish a snapshot to the Apache release repo.
```

Comment:
Sorry what's the meaning of this step, I don't remember I have a such step.

Reply:
This is how we describe it before: https://github.com/apache/spark-website/pull/157/files#diff-12257e3523e2ce22fccd80687597e7cdL94

+1 that "Apache staging Maven repo" is cleaner, I'll update soon